citesphere/src/main/webapp/WEB-INF/views/auth/group/items.jsp

```diff
@@ -146,7 +146,25 @@ $(function() {
 </div>
 
 <div class="col-md-2">
-<p class="lead">Collections</p>
+<c:set var = "columnString" value = ""/>
+<c:forEach items="${columns}" var="column" varStatus="loop">
+   <c:set var = "columnString" value = "${columnString}${column}"/>
+   <c:if test="${!loop.last}">
+   		<c:set var = "columnString" value = "${columnString},"/>
+   </c:if>
+</c:forEach>
+<p class="lead">Collections &nbsp;&nbsp;
+<c:choose>
+    <c:when test="${collectionId!=null}">
+        <a href="<c:url value="/auth/group/${zoteroGroupId}/collection/${collectionId}/items/sync?page=${currentPage}&sort=${sort}&columns=${columnString}" />"><i
+		class="fas fa-sync" title="Sync Citation"></i></a>
```

Comment:
should be "Sync Page" and let's surround the i-element with a  `small` (same below)